2996_reflection/reflection.md

```diff
@@ -5734,6 +5736,38 @@ int a = fn<^^NS>();
 :::
 :::
 
+### 13.8.3.7 [temp.dep.concept] Dependent concepts {-}
+
+Add a new section to cover dependent concepts.
+
+::: std
+:::addu
+**Dependent concepts   [temp.dep.concept]**
+
+[1]{.pnum} A concept is dependent if it is designated by a dependent `$splice-specifier$`. A `$concept-name$` is dependent if its lookup context is dependent.
```

Comment:
I think in light of P2841 being in flight we might want to make the term more specific here. Like maybe say a concept is *splice-dependent* if that, and a program is ill-formed we name/designate a splice-dependent concept?